README.md

```diff
@@ -1,16 +1,204 @@
+- The **Guides** section helps a developer get comfortable with individual Appwrite services. Guides are structured to walk a developer through features of an Appwrite service. The topics should be ordered by level of complexity, where complex topics are positioned later into a guide.
+- The **REST API** section helps a developer understand details of individual endpoints. Additions to REST API documentation should remain focused on the API endpoint itself. Avoid adding information about related endpoints and services that are not integral to undestanding documented endpoint. Code example changes should be submitted to the [SDK Generator repo](https://github.com/appwrite/sdk-generator).
+- The **Advanced** section helps a developer learn more complicated concepts like pagination or using permissions. If your proposed addition are not a core component of using a service or requires lengthy explanation, consider adding them to **Advanced**.
+- **External Content** like blog posts and videos are used for documenting examples or integrating Appwrite with third party services. If your proposed addition involves content that are not focused on specific ways of using Appwrite, create a piece of external content. Add the content to the [Awesome Appwrite](https://github.com/appwrite/awesome-appwrite) catalog. The Appwrite team will also be glad to share the external content on Discord and on Appwrite's social accounts.
+
+As a general note, if a behavior is excessively difficult to explain or feels awkwardly complex to document, there is a chance that it is also a design/product issue or bug. Consult Appwrite maintainers and open an issue.
 
 ## Style Guidelines
+When contributing to Appwrite's documentation, use these guidelines to format your contributions.
+### Page Hierarchy
+Titles and subtitles should follow a logical hierarchy (h1-h6) using the [AP style of headline capitalization](https://capitalizemytitle.com/style/AP/). Page headings and subheadings should be hyperlinked for ease of reference and have an appropriate ID. For examples:
+```html
+<h2><a href="/docs/command-line#installation" id="installation">Installation</a></h2>
+```
+
+### Naming Files
+File names should reflect it's content. In general, use dash separated file names that reference the title of the page. For example "Documentation Guidelines" could be "documentation-guidelines".
+### Internal and External Links
+Internal links that direct to any link under [https://appwrite.io](/docs/command-line#installation) should be relative and precise. For example, when referencing the Appwrite Documentation page, use the `<a href="/docs">Documentation</a>`. When referencing a specific section of a page, link to the section heading where possible. For example `<a href="/docs/databases#databases">Create Your Databases</a>`.
```

Comment:
I get what you mean, but I don't think these links are called `relative`. Relative would be `../docs/command-line`. Maybe there is a batter word for internal VS external links?

Reply:
https://www.w3.org/TR/WD-html40-970917/htmlweb.html

I believe this is in fact relative:

><img width="1239" alt="Screen Shot 2022-08-02 at 5 30 27 PM" src="https://user-images.githubusercontent.com/29069505/182476880-9c02a396-47e5-4fdc-b81c-dee0a497b32d.png">